google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import six
+
+class pandas_df():
```

Comment:
I see no reason for this to be a class.

Reply:
Modified code to remove class and keep just the method